[2021-12-26 19:03:23,309] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-25T09:00:00+00:00 [queued]>
[2021-12-26 19:03:23,324] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-25T09:00:00+00:00 [queued]>
[2021-12-26 19:03:23,324] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-12-26 19:03:23,324] {taskinstance.py:881} INFO - Starting attempt 3 of 3
[2021-12-26 19:03:23,325] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-12-26 19:03:23,340] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-12-25T09:00:00+00:00
[2021-12-26 19:03:23,344] {standard_task_runner.py:54} INFO - Started process 36834 to run task
[2021-12-26 19:03:23,382] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-12-25T09:00:00+00:00', '--job_id', '18', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmp9btns0nb']
[2021-12-26 19:03:23,383] {standard_task_runner.py:78} INFO - Job 18: Subtask twitter_aluraonline
[2021-12-26 19:03:23,417] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-25T09:00:00+00:00 [running]> on host note-cristian-agriness
[2021-12-26 19:03:23,455] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-12-26 19:03:23,457] {twitter_hook.py:38} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-25T09:00:00.00Z&end_time=2021-12-26T09:00:00.00Z
[2021-12-26 19:03:23,483] {logging_mixin.py:112} WARNING - /home/cristian/Desktop/datapipeline/.env/lib/python3.6/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2021-12-26 19:03:23,774] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211225T090000, start_date=20211226T220323, end_date=20211226T220323
[2021-12-26 19:03:28,296] {local_task_job.py:102} INFO - Task exited with return code 0
